source/features/releases-tab.tsx

```diff
@@ -15,14 +15,14 @@ import {buildRepoURL, getRepo} from '../github-helpers';
 
 const getCacheKey = (): string => `releases-count:${getRepo()!.nameWithOwner}`;
 
-function parseCountFromDom(): number {
+async function parseCountFromDom(): Promise<number> {
 	const releasesCountElement = select('.numbers-summary a[href$="/releases"] .num');
 	if (releasesCountElement) {
 		return looseParseInt(releasesCountElement);
 	}
 
-	// In "Repository refresh" layout, look for the releases link in the sidebar
-	const moreReleasesCountElement = select('[href$="/tags"] strong');
+	// In "Repository refresh" layout, look for the tags link in the header
+	const moreReleasesCountElement = await elementReady('[href$="/tags"] strong');
```

Comment:
The bug was caused by this selector. It currently targets the `xx tags` link in the homepage header:
![image](https://user-images.githubusercontent.com/46634000/111513796-7467ea00-8751-11eb-9fc0-dd6b0197dc1e.png)

But since it's lazy-loaded, sometimes it doesn't exist yet and thus `parseCountFromDom()` returns `0`, which in turn causes the feature to give up.

Reply:
Keep in mind that [by default](https://github.com/sindresorhus/element-ready#stopondomready) `elementReady` does not work for lazy-loaded elements and the option to do so is unsafe. You'll have to rely on whatever load event they fire or use an observer: simplified, oneMutation or (ideally not) selector-observer